nrf_dm/include/nrf_dm.h

```diff
@@ -249,19 +249,21 @@ typedef struct {
 #define NRF_DM_DEFAULT_TX_POWER RADIO_TXPOWER_TXPOWER_0dBm
 #endif
 
+#define NRF_DM_DEFAULT_ANTENNA_COMP (NRF_DM_ANTENNA_COMP_1_1)
 /**
  * @brief Default configuration define, note that both role and access_address
  * needs to be configured even when using the default config.
  */
-#define NRF_DM_DEFAULT_CONFIG                                                              \
-	((nrf_dm_config_t){                                                                    \
```

Comment:
what changed here? Is using whitespaces instead of tabs?

Reply:
Spaces on both. Just aligned the backslashes